elastic/datadog_checks/elastic/elastic.py

```diff
@@ -40,6 +41,13 @@ def __init__(self, name, init_config, instances):
         # Host status needs to persist across all checks
         self.cluster_status = {}
 
+        if self.instance.get('auth_type') == 'aws' and self.instance.get('url'):
+            self.HTTP_CONFIG_REMAPPER = self.HTTP_CONFIG_REMAPPER.copy()
+            self.HTTP_CONFIG_REMAPPER['aws_host'] = {
+                'name': 'aws_host',
+                'default': urlparse(self.instance['url']).hostname,
+            }
```

Comment:
Why not move this logic to `RequestsWrapper.__init__` in the first place ?

It's related to how `aws` auth type, not specific to `elastic.

---

Side effect, we won't need to `self.HTTP_CONFIG_REMAPPER = self.HTTP_CONFIG_REMAPPER.copy()` anymore.